apps/test/beeDrawingTest.js

```diff
@@ -1,25 +1,25 @@
-var jsdom = require('jsdom').jsdom;
+//var jsdom = require('jsdom').jsdom;
 var testUtils = require('./util/testUtils');
 
 testUtils.setupLocales();
 
 function setGlobals() {
-  var html = '<html><head></head><body><div id="svgMaze"><div class="pegman-location"></div></div></body></html>';
-  global.document = jsdom(html);
-  global.window = global.document.parentWindow;
-  global.Blockly = {
-    SVG_NS: "http://www.w3.org/2000/svg"
-  };
+  document.body.innerHTML = '<div id="svgMaze"><div class="pegman-location"></div></div>';
+  //global.document = jsdom(html);
+  //global.window = global.document.parentWindow;
+  //global.Blockly = {
+  //  SVG_NS: "http://www.w3.org/2000/svg"
+  //};
 
   assert(document, 'have a document');
   assert(document.getElementById('svgMaze'), 'document has an svgMaze');
   assert(document.getElementsByClassName('pegman-location').length, 1);
 }
 
-var Bee = testUtils.requireWithGlobalsCheckBuildFolder('maze/bee');
-var BeeItemDrawer = testUtils.requireWithGlobalsCheckBuildFolder('maze/beeItemDrawer');
-var DirtDrawer = testUtils.requireWithGlobalsCheckBuildFolder('maze/dirtDrawer');
-var cellId = testUtils.requireWithGlobalsCheckBuildFolder('maze/mazeUtils').cellId;
+var Bee = require('@cdo/apps/maze/bee');
+var BeeItemDrawer = require('@cdo/apps/maze/beeItemDrawer');
+var DirtDrawer = require('@cdo/apps/maze/dirtDrawer');
+var cellId = require('@cdo/apps/maze/mazeUtils').cellId;
```

Comment:
yeah, this is a pattern i discovered because shifting the test suite to Browserify meant that `require(dynamic_string)` calls no longer work, and this was the recommended approach to resolving local modules like these in a non-relative manner.